content/docs/command-reference/remote/modify.md

```diff
@@ -585,12 +585,26 @@ for a full guide on using Google Drive as DVC remote storage.
   $ dvc remote modify myremote gdrive_client_secret 'client-secret'
   ```
 
-- `gdrive_user_credentials_file` - path where DVC stores OAuth credentials to
-  access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
+- `profile` - name used to cache OAuth credentials. Useful to avoid passing the
+  wrong credentials, for example when using multiple GDrive remotes. The default
```

Comment:
hmm, I think the previous version was a bit better, more explicit ... how was it phrased? it not about passing wrong credentials only, it's about also rewriting wrong credentials

we are overcomplicating this, Jorge ...

Reply:
> how was it phrased?

"if you are using multiple users you might want to
  use ‘profile’ to avoid accidentally using someone else’s cached credentials"

- [original commit](https://github.com/iterative/dvc.org/pull/3795/commits/57dc421526c6fd650aacac5d3372e0b3dd523ae1#diff-a99e08a2de18082aa1adc45807e98187fe62f245a715f12cd08ec703823c9273R606-R608)

> it's about also rewriting...

Right, also only [if it's the same client](https://github.com/iterative/dvc.org/pull/3795#pullrequestreview-1055034448) I guess. Fixed.